src/Sulu/Bundle/ContactBundle/Api/Contact.php

```diff
@@ -851,12 +851,7 @@ public function getGender()
     #[Groups(['fullContact'])]
     public function getAccount()
     {
-        $mainAccount = $this->entity->getMainAccount();
-        if (!\is_null($mainAccount)) {
-            return new Account($mainAccount, $this->locale);
-        }
-
-        return;
+        return $this->entity->getMainAccount()?->getId();
```

Comment:
this method should still return the account.

Create a new `getAccountId` method for the admin controller serialization.